From 4cc5657247183a0bc3b0969beeaea9acddb09d22 Mon Sep 17 00:00:00 2001
From: Daniel Stenberg <daniel@haxx.se>
Date: Wed, 22 May 2024 08:43:43 +0200
Subject: [PATCH] docs/Makefile.am: make curl-config.1 install

on "make install" like it should

Follow-up to 60971d665b9b1df87082

Closes #13741
---
 docs/Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/docs/Makefile.am b/docs/Makefile.am
index 83f5b0c461cc0f..e9ef6284860555 100644
--- a/docs/Makefile.am
+++ b/docs/Makefile.am
@@ -28,6 +28,7 @@ if BUILD_DOCS
 # if we disable man page building, ignore these
 MK_CA_DOCS = mk-ca-bundle.1
 CURLCONF_DOCS = curl-config.1
+man_MANS = curl-config.1
 endif
 
 CURLPAGES = curl-config.md mk-ca-bundle.md
